android/src/main/java/io/rownd/android/views/RowndWebView.kt

```diff
@@ -259,6 +259,7 @@ class RowndJavascriptInterface(
                         )
                     )
                 )
+                parentWebView.rowndClient.signInRepo.reset()
```

Comment:
This PR looks good to me. This line is the only thing I'm struggling to wrap my head around. Can you explain what this is doing and why it's needed?

Reply:
Should the Hub's saved sign-in method be sent through this interface so that it aligns with the internal swift state?

Reply:
At this time only the Hub needs to know the latest Sign in method to then reorganize the SignIn Modal.

I could see us moving to a point where in iOS/Android we'd automatically trigger the users last sign-in method. Whether it be in the Hub (Email/Phone) or outside of it (Google/Apple). If we're gonna go that route we'd need the Swift state to be up to date with last sign in.

Reply:
I'm tracking now. That makes sense. Thanks for the explanation.